hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ReplicationManager.java

```diff
@@ -154,6 +151,9 @@ public synchronized void start() {
     if (!running) {
```

Comment:
Here we need to check isRunning() method.
Take an example:
call start, stop, (but it has not stopped completely, calling start immediately we try to start again if we only check running flag.

Reply:
Makes sense, will change it.